conanfile.py

```diff
@@ -0,0 +1,23 @@
+from conans import ConanFile
+from conans.tools import download, untargz
+import os
+
+class TypeSafe(ConanFile):
+    name = 'type_safe'
+    url  = 'https://foonathan.github.io/blog/2016/10/11/type-safe.html'
+    version = '0.1'
+    requires = 'debug_assert/1.3@Manu343726/testing'
```

Comment:
I'm guessing I have to update this one everytime I release a new version and/or update debug_assert?